phpBB/includes/acp/acp_attachments.php

```diff
@@ -225,6 +225,9 @@ function main($id, $mode)
 					if (in_array($config_name, array('attachment_quota', 'max_filesize', 'max_filesize_pm')))
 					{
 						$size_var = $request->variable($config_name, '');
+
+						$config_value = (int) $config_value;
```

Comment:
I suggest make this type casting on line `223` as ; 
Line `223` : `$this->new_config[$config_name] = $config_value = $cfg_array[$config_name];`
to : 
Line `223` : `$this->new_config[$config_name] = $config_value = (int) $cfg_array[$config_name];`

Best regards 👍

Reply:
No as that will also effect the upload directory